requirements.txt

```diff
@@ -1,11 +1,11 @@
 numpy >= 1.18
-h5py<2.11.0,>=2.10.0
-scikit-image
+h5py>=2.10.0
+scikit-image==0.17.2
```

Comment:
Scikit image must be <=0.17.2 .. I will get another PR for this soon, but this is a good first fix.